### PR TITLE
docs: move docs, update/rename registry schema

### PR DESCRIPTION
**Pull request by kitsonk**

### CODE REVIEW

File: schemas/registry-completions.schema.json [hunk 1]
```diff
@@ -11,9 +11,11 @@
   "properties": {
     "version": {
       "description": "The version of the schema document.",
-      "type": "number",
+      "type": "integer",
```
Comment (dsherret) [inline schemas/registry-completions.schema.json:14]: I did not know there was an integer type in json schema.

File: schemas/registry-completions.schema.json [hunk 2]
```diff
@@ -67,16 +69,25 @@
...
+            "https://example.com/api/packages/${package}",
+            "https://example.com/api/packages/${package}/${{version}}",
+            "/api/packages/${package}/${{version}}/${path}"
           ]
         }
       }
     }
   }
-}
+}
```
Comment (dsherret) [inline schemas/registry-completions.schema.json:93]: I wonder what is removing the final newlines. Not that it really matters, but seems like we don't have code formatting on this project?

**Review by dsherret — APPROVED** [review]
LGTM

File: schemas/registry-completions.schema.json (hunk 1 above)
Comment (kitsonk) [reply schemas/registry-completions.schema.json:14]: I didn't either until I looked up how to constrain the value.

File: schemas/registry-completions.schema.json (hunk 2 above)
Comment (kitsonk) [reply schemas/registry-completions.schema.json:93]: Having an old formatter configured in my IDE for JSON files and not running `npm fmt` before comitting!

**Pull request merged**